sklearn/cross_decomposition/pls_.py

```diff
@@ -74,7 +75,8 @@ def _nipals_twoblocks_inner_loop(X, Y, mode="A", max_iter=500, tol=1e-06,
         if np.dot(x_weights_diff.T, x_weights_diff) < tol or Y.shape[1] == 1:
             break
         if ite == max_iter:
-            warnings.warn('Maximum number of iterations reached')
+            warnings.warn('Maximum number of iterations reached',
```

Comment:
you can probably test this by setting ``max_iter=1``.

Reply:
I've added a test with low `max_iter` and very low `tol` that triggers this `ConvergenceWarning`. The other two warnings about unchanged Y residuals or X scores are not tested, though.